config/7.12.0/pim/process-migration.yaml

```diff
@@ -61,23 +64,27 @@ processMigration:
                 livenessProbe:
                   failureThreshold: 3
                   httpGet:
-                    path: /health
+                    path: /q/health/live
                     port: 8080
                     scheme: HTTP
                   initialDelaySeconds: 180
                   periodSeconds: 15
                   successThreshold: 1
                   timeoutSeconds: 2
                 env:
-                  - name: JBOSS_KIE_ADMIN_USER
-                    value: "[[.AdminUser]]"
-                  - name: JBOSS_KIE_ADMIN_PWD
-                    value: "[[.AdminPassword]]"
-                  - name: JBOSS_KIE_EXTRA_CONFIG
-                    value: "/opt/rhpam-process-migration/config/project-overrides.yml"
+                  #[[if .ProcessMigration.ExtraClassPath]]
```

Comment:
this env variable is not used

Reply:
see my other comment, let me know if that makes sense to you.

Reply:
total sense, thanks